### ISSUE TITLE
Set password from Python?

### ISSUE THREAD

**Issue by m01** [RoButton/switchbotpy#15]
Does anyone know which BLE command is used to set the password? I'd like to avoid using the app.

**Comment by pelag0s** [comment]
Yes, this would be helpful! Especially as in the latest app version (at least on Android) they capped the password length to 4 characters. Which is a security nightmare to say the least.
